src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,170 @@
+                                    title={item.title}
+                                    subtitle={item.subtitle}
+                                    shouldPlaceSubtitleBelowSwitch
+                                    wrapperStyle={styles.mv3}
+                                    isActive={item.isActive}
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                pendingAction={pendingFields?.collectionAccountID}
+                                isActive={isSyncReimbursedSwitchOn}
+                                onToggle={() =>
+                                    Connections.updatePolicyConnectionConfig(
+                                        policyID,
+                                        CONST.POLICY.CONNECTIONS.NAME.QBO,
+                                        CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID,
+                                        isSyncReimbursedSwitchOn ? '' : selectedAccount,
+                                    )
+                                }
+                            />
+
+                            {collectionAccountID && (
+                                <>
+                                    <OfflineWithFeedback pendingAction={pendingFields?.collectionAccountID}>
+                                        <MenuItemWithTopDescription
+                                            shouldShowRightIcon
+                                            title="Croissant Co Payroll Account" // TODO: set to the current selected value
+                                            description={translate('workspace.qbo.advancedConfig.qboAccount')}
+                                            wrapperStyle={[styles.sectionMenuItemTopDescription]}
+                                            onPress={waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_ACCOUNT_SELECTOR.getRoute(policyID)))}
+                                            errorText={errors?.collectionAccountID}
+                                            brickRoadIndicator={errors?.collectionAccountID ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
```

Comment:
```suggestion
                                            error={errors?.reimbursementAccountID}
                                            brickRoadIndicator={errors?.reimbursementAccountID ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
```

- reimbursementAccountID is what this menu item related to
- Use error instead of errorText. errorText is buggy (ref https://github.com/Expensify/App/pull/39191#discussion_r1571107694)